### ISSUE TITLE
Checkbox Example (Mixed State) intermediate state only through colour

### ISSUE THREAD

**Issue by MichielBijl** [w3c/aria-practices#295]
The visual difference between when “All condiments” checkbox has `aria-checked="mixed"` and when it has `aria-checked="true"` is a change in background colour. This should be communicated through a change in icon. Convention on macOS is a dash inside the checkbox instead of a checkmark.

[Tri-state checkbox example page](http://w3c.github.io/aria-practices/examples/checkbox/checkbox-2/checkbox-2.html)

**Comment by MichielBijl** [comment]
New design implemented in: https://github.com/w3c/aria-practices/commit/4781f284f506d74c0baf226de7eab13c93117052

- All graphics are generated with CSS generated content (they’re not spoken by AT, don’t you worry!)
- States are easily differentiated in default mode and Windows High Contrast Mode
